tone-analyzer/src/main/java/com/ibm/watson/developer_cloud/tone_analyzer/v3/ToneAnalyzer.java

```diff
@@ -107,4 +110,21 @@ public ToneAnalyzer(String versionDate, String username, String password) {
 
     return createServiceCall(requestBuilder.build(), ResponseConverterUtils.getObject(ToneAnalysis.class));
   }
+
+  /**
+   * Analyzes the "tone" of a list of utterances in a conversation. The text is analyzed from several chat tones, and
+   * confidence score is given back for tones which are present in the text
+   *
+   * @param jsonText The text in JSON format to analyze
+   * @return the {@link UtterancesTone} with the response
+   */
+  public ServiceCall<UtterancesTone> getChatTone(String jsonText) {
```

Comment:
@shubhanjan99 we don't use a string with JSON as parameters.  Can you create the Java classes representing the request? We probably want a Builder pattern also.

cc: @mkistler